packages/shylu/shylu_dd/bddc/src/ProblemMakerBDDC.h

```diff
@@ -52,16 +52,30 @@
 #include <algorithm>
 #include <vector>
 #include <complex>
-#include <assert.h>
 
+#ifdef HAVE_SHYLU_DDBDDC_METIS
 #include "metis.h"
+#endif
 
 #include "Teuchos_RCP.hpp"
 #include "Teuchos_ParameterList.hpp"
 #include "Teuchos_BLAS.hpp"
 #include "Teuchos_LAPACK.hpp"
 
 #include "shylu_enumsBDDC.h"
+#include "shylu_errorBDDC.h"
+
+#ifdef _OPENMP
+#include "omp.h"
+#else
+#define omp_get_max_threads() 1
```

Comment:
I shouldn't speak for Clark, but I think the point of this was a quick hack to make OpenMP code compile without OpenMP support.  It would be better to encapsulate something like this in a function, if the code isn't suitable for translation to Kokkos.

Reply:
The subject code was not needed and was removed.